docs/kilnjs.md

```diff
@@ -0,0 +1,220 @@
+---
+id: kilnjs
+title: Kilnjs
+sidebar_label: Kilnjs
+---
+
+---
+
+Kilnjs is an optional Javascript file that transforms the schema.yaml file into a dynamic Javascript Object that affects how Kiln interacts with components that use the schema.  With it, events can be attached to the inputs that are used by kiln to edit components.  These events can be used to hide or reveal different fields in the schema, update the values of those fields, make api calls, subscribe to Vuex mutations, and just about any other action allowed by JavaScript.
+
+Using Kilnjs, you can remove the logic from the schema.yaml file and move it into JavaScript, keeping the schema file for presentational information.
+
+To take advantage of the power that Kilnjs provides you need to create a file called kiln.js within a component folder, in the same location where you place the component's schema, model, and client files. At a minimum, it should look like this.
+```
```

Comment:
Missing language definition for styles and format

```suggestion
```js
```

Reply:
👍